demo/tekton-demo-setup/operator_tekton_demo_values.yaml.sample

```diff
@@ -0,0 +1,54 @@
+kind: Pelorus
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+metadata:
+  name: pelorus-sample
```

Comment:
`pelorus-tekton-demo` or something like this?

Reply:
meh doesn't matter